### ISSUE TITLE
Tooltip box and text not showing up properly

### ISSUE THREAD

**Issue by suelspahiu** [recharts/recharts#122]
Thank you for recharts!
I've got a simple example going and the tooltip box is too small to show all the data.
Also the text is inline instead of in 3 separate lines.
I'm using es6, babel, webpack.

![image](http://i.imgur.com/DESUdkk.png?1)

Sorry, I couldn't make it work on JSFiddle. Here's my code:

```
import React from 'react';
import { ComposedChart, Line, Area, Bar, XAxis, YAxis, CartesianGrid, Tooltip, Legend } from 'recharts';
const data = [{name: 'Page A', uv: 590, pv: 800, amt: 1400},
              {name: 'Page B', uv: 868, pv: 967, amt: 1506},
              {name: 'Page C', uv: 1397, pv: 1098, amt: 989},
              {name: 'Page D', uv: 1480, pv: 1200, amt: 1228},
              {name: 'Page E', uv: 1520, pv: 1108, amt: 1100},
              {name: 'Page F', uv: 1400, pv: 680, amt: 1700}];

export default class LineBarAreaComposedChart extends React.Component {
	render () {
  	return (
    	<ComposedChart width={600} height={400} data={data}
            margin={{top: 20, right: 80, bottom: 20, left: 20}}>
          <XAxis dataKey="name" label="Pages"/>
          <YAxis label="Index"/>
          <Tooltip/>
          <Legend/>
          <CartesianGrid stroke='#f5f5f5'/>
          <Area type='monotone' dataKey='amt' fill='#8884d8' stroke='#8884d8'/>
          <Bar dataKey='pv' barSize={20} fill='#413ea0'/>
          <Line type='monotone' dataKey='uv' stroke='#ff7300'/>
       </ComposedChart>
    );
  }
}
```

Please advise.

**Comment by xile611** [comment]
@suelspahiu It works fine in [JSFiddle](http://jsfiddle.net/sn2j92kq/).  It seems that you may have some css like:

```
li {
   float: left;
}
```

Because we use `li` to write the texts in tooltip.

**Comment by suelspahiu** [comment]
@xile611 Thanks, fixed!
Keep up the good work.

**Comment by suelspahiu** [comment]
@xile611 I've got another question, sorry.
Is it possible to have the line segments between data points to be drawn in a straight fashion, without curves? I've tried the "type" property but no luck.

I mean: 
![screen shot 2016-06-23 at 5 27 13 pm](https://cloud.githubusercontent.com/assets/9028592/16320822/c647963c-3967-11e6-8c0f-3df67b35997e.png)


instead of:

![screen shot 2016-06-23 at 5 28 18 pm](https://cloud.githubusercontent.com/assets/9028592/16320854/fb979bde-3967-11e6-8e5e-030b9ec07781.png)

**Comment by xile611** [comment]
Yeah, you can set the value of `type` to 'linear' or remove `type`, because the default value of `type` is `linear`.
```
<Line type="linear" dataKey="any"/>
```

or 

```
<Line dataKey="any"/>
```

You can look at [api](http://recharts.org/api#Line) for more infromation.

**Comment by suelspahiu** [comment]
@xile611 Somehow I totally missed that. Thanks!
I have 1 more problem: I'm trying to make a slider where the actual slider is static(just a data point) but the chart shown will be something like this:

![screen shot 2016-06-24 at 11](https://cloud.githubusercontent.com/assets/9028592/16342080/52ec55be-39fe-11e6-8430-7ac00a7ac9c6.png)

I've played around with layout=vertical with a combination of area and a bar, area and line, line and bar but for some reason I can't quite get the layout right.
So my question is: 
1- Can something like this be done with recharts
2- How hard would it be to create a new "shape" of dot to match the 
![screen shot 2016-06-24 at 11-1](https://cloud.githubusercontent.com/assets/9028592/16342267/34e38a3c-39ff-11e6-8d7b-6e41028d958b.png)


Sorry for extra questions but I love the library and don't want to use something else if it's possible.
Thank you again!

**Comment by xile611** [comment]
@suelspahiu 

You can see [this example](http://jsfiddle.net/ffh6nobj/) to determine whether Recharts meet your requirement.

**Comment by suelspahiu** [comment]
@xile611 
This looks great!
I was thinking about this and there's another part in my application that I'd like to have an active slider, basically exactly like the one above but movable, where the user can move the slider back and forth. How would I go about that?
Sorry to ask a million questions. You're the greatest.

**Comment by suelspahiu** [comment]
Hello again @xile611 ,
Would you please let me know if there's a way to change the font used throughout the graphs? I've looked for a while and cant seem to find anything. Thanks!

**Comment by xile611** [comment]
@suelspahiu 
There are two ways to change the font of texts in the chart.
One is through css like the following:

```css
// set the font of all the texts in the chart
.recharts-wrapper { 
  font-size: 16px;
  font-weight: bold;
}
// set the style of the ticks of y-axis
.recharts-wrapper .recharts-y-axis .recharts-cartesian-axis-tick text {
  fill: red;
  font-size: 16px;
}
``` 

Another is through attributes of  components:

```jsx
<LineChart width={600} height={300} data={data} style={{fontSize: 18}}
            margin={{top: 5, right: 30, left: 20, bottom: 5}}>
  <XAxis dataKey="name" fontSize={16} fontFamily="sans-serif" />
  <YAxis/>
  <CartesianGrid strokeDasharray="3 3"/>
  <Tooltip/>
  <Legend />
  <Line type="monotone" dataKey="pv" stroke="#8884d8" activeDot={{r: 8}}/>
  <Line type="monotone" dataKey="uv" stroke="#82ca9d" />
</LineChart>
```
The styling of svg elements is just similar to the styling of dom nodes. You can check [this document](https://www.w3.org/TR/SVG/styling.html) for detail.


And for the problem of ResponsiveContainer, I'll fix this bug soon. Thank you!

**Comment by suelspahiu** [comment]
@xile611 Thank you ! Please let me know when it's fixed!
Take care.

**Comment by xile611** [comment]
@suelspahiu  Fixed in version 0.13.4

**Comment by suelspahiu** [comment]
@xile611 Thank you.
